### ISSUE TITLE
Formatted (JSON) logs have duration with no unit

### ISSUE THREAD

**Issue by arielshaqed** [treeverse/lakeFS#3744]
When collecting logs from JSON format into Grafana we get logs such as these (reformatted for beauty and legibility):

```json
{
  "file": "usr/local/go/src/net/http/server.go:2047",
  "func": "net/http.HandlerFunc.ServeHTTP",
  "host": "treeverse.eu-central-1.lakefscloud.io",
  "level": "info",
  "log_audit": "API",
  "method": "GET",
  "msg": "HTTP call ended",
  "path": "/api/v1/repositories/something",
  "request_id": "a296ebfe-eb9f-4856-b59f-cd82b7e1cf36",
  "sent_bytes": 155,
  "service_name": "rest_api",
  "status_code": 200,
  "time": "2022-07-17T20:53:23Z",
  "took": 295165694
}
```

The duration field "took" is a pure number with no units.  I _suspect_ that `time.Duration` is an integer, so it gets JSON-serialized.  When logs are output in text format, it _does_ get stringified so it gets a number.

As a temporary workaround we could figure out the units and use those for now.

**Comment by arielshaqed** [comment]
@idanovo this is what we saw on Grafana. :-(

**Comment by arielshaqed** [comment]
> @idanovo this is what we saw on Grafana. :-(

... About _time displayed in logs_.  This should **not** affect the time displayed from metrics in any way whatsoever, it's a separate reporting path.

**Comment by arielshaqed** [comment]
High priority because affects any perf debugging.

**Comment by arielshaqed** [comment]
This also occurs in the "duration" field of database logs :-(

**Comment by arielshaqed** [comment]
Actually fixed by #6934 :-)
